app/views/auth/registrations/_sessions.html.haml

```diff
@@ -19,10 +19,10 @@
         %td
           %samp= session.ip
         %td
-          - if request.session['auth_id'] == session.session_id
+          - if request.cookies.signed['_session_id'] == session.session_id
```

Comment:
Cannot we use `ApplicationController#current_session` or something for this, instead accessing cookie directly?